docs/connect-storefront-to-platform-v3.md

```diff
@@ -0,0 +1,40 @@
+# Connect Storefront to Platform
+
+* Deploy the latest storefront version as described [there](https://virtocommerce.com/docs/vc2devguide/deployment/storefront-deployment)
+
+Note if you deploy the Storefront on local environment with the Platform you should host it with IIS
```

Comment:
Note, if Platform and Storefront are deployed in the on-premises environment, they should be hosted in IIS.

Comment: but why IIS???

Reply:
Note, if Platform and Storefront are deployed in the same on-premises environment, Storefront should be deployed on different port then Platform. You can do it by `dotnet run CLI`

```console
dotnet VirtoCommerce.Storefront.dll --urls=http://localhost:5002/
```

or host Storefront in IIS.